app/packages/looker/src/worker/resolve-color.ts

```
@@ -0,0 +1,60 @@
+
+export interface ResolveColor {
+  key: string | number;
+  seed: number;
+  color: string;
+}
+
+export default (): [
+  (pool: string[], seed: number, key: string | number) => Promise<string>,
+  (result: ResolveColor) => void
+] => {
+  const cache = new LRUCache<number, {}>({
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)